### PR TITLE
fix divide by zero on nu_org and kappa

### PR DESCRIPTION
**Pull request by claresinger**
fix divide by zero error when aerosol is 100% organic OR 100% inorganic. define new func that handles these edge cases.

solution to issue https://github.com/atmos-cloud-sim-uj/PySDM/issues/756

### CODE REVIEW

**Comment by slayoo** [comment]
@claresinger to run pylint locally before committing, you can use the commands from the Github Actions workflow file:
https://github.com/atmos-cloud-sim-uj/PySDM-examples/blob/main/.github/workflows/pylint.yml#L30
https://github.com/atmos-cloud-sim-uj/PySDM-examples/blob/main/.github/workflows/pylint.yml#L34
i.e.:
```bash
pylint --disable=fixme,too-many-function-args,unsubscriptable-object,consider-using-with,protected-access,too-many-statements,too-many-public-methods,too-many-branches,duplicate-code,invalid-name,missing-function-docstring,missing-module-docstring,missing-class-docstring,too-many-locals,too-many-instance-attributes,too-few-public-methods,too-many-arguments $(git ls-files '*.py')

nbqa pylint --unsafe-load-any-extension=y --disable=fixme,duplicate-code,invalid-name,trailing-whitespace,line-too-long,missing-function-docstring,wrong-import-position,missing-module-docstring,wrong-import-order,ungrouped-imports,no-member,too-many-locals,redefined-outer-name $(git ls-files '*.ipynb')
```

HTH, thanks!

**Comment by claresinger** [comment]
Thank you Sylwester!! This will make my life so much better 😄

File: PySDM_examples/Lowe_et_al_2019/aerosol.py [hunk 1]
```diff
@@ -98,7 +105,8 @@ class Aerosol:
 @strict
 class AerosolMarine(Aerosol):
     Aitken = {'palmitic': .2, 'SOA1': 0, 'SOA2': 0, '(NH4)2SO4': .8, 'NH4NO3': 0, 'NaCl': 0}
-    Accumulation = {'palmitic': .2, 'SOA1': 0, 'SOA2': 0, '(NH4)2SO4': 0, 'NH4NO3': .8, 'NaCl': 0}
+    #Accumulation = {'palmitic': .2, 'SOA1': 0, 'SOA2': 0, '(NH4)2SO4': 0, 'NH4NO3': .8, 'NaCl': 0}
```
Comment (slayoo) [inline PySDM_examples/Lowe_et_al_2019/aerosol.py:108]: @claresinger is this change intentional?

Comment (claresinger) [reply PySDM_examples/Lowe_et_al_2019/aerosol.py:108]: Oh sorry! I was just confirming that it works for f_org = 0 and f_org = 1 now with this if-else formulation and forgot to delete this change.
